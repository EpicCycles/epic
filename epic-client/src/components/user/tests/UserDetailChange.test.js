import React from "react";
import UserDetailChange from "../UserDetailChange";

test("displays correctly", () => {
    const component = shallow(<UserDetailChange changeUserData={jest.fn()} />)
    expect(component).toMatchSnapshot();
});
test("displays correctly with a user", () => {
    const user = {user: {
        username: 'fred',
        first_name: 'Frederick',
        last_name: 'Jones,',
        email: 'fred@fred.jones.org'
    }}
    const component = shallow(<UserDetailChange
        changeUserData={jest.fn()}
    user={user}/>);
    expect(component).toMatchSnapshot();
});