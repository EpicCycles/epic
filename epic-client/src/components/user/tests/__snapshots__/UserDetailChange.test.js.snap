// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays correctly 1`] = `
<div
  className="red error"
>
  You need to log in,
</div>
`;

exports[`displays correctly with a user 1`] = `
<div
  className="red error"
>
  You need to log in,
</div>
`;
